requirements.txt

```diff
@@ -1,2 +1,4 @@
 git+https://github.com/aeroo/aeroolib.git
 git+https://github.com/aeroo/currency2text.git
+# use this genshi version to fix error when, for eg, you send arguments like "date=True" check this https://genshi.edgewall.org/ticket/600
+git+https://github.com/edgewall/genshi@stable/0.7.x
```

Comment:
Newer version to fix ror with genshi if, for eg. you call "formatlang(o.date, date=True)" (the issue was with date=True)